lang/en.json

```diff
@@ -777,67 +796,67 @@
-        "WindowLoadedTriggerDescription": "Triggered when the page is fully loaded including all images etc.",
-        "WindowLoadedTriggerHelp": "This stage of loading a web page, is when the page is fully loaded according to the browser. In most cases this means that also all images are loaded, and all styles are applied etc. This trigger is always triggered at some point after the \"DOM Ready\".",
-        "WindowLoadedTriggerName": "Window Loaded",
-        "WindowUnloadTriggerDescription": "Triggered just before the browser window is closed or when the user navigates to a different page.",
-        "WindowUnloadTriggerHelp": "This trigger identifies the event when a user is actually closing the current page compared to the \"Window Leave\" trigger which triggers when the user is about to leave your page. Either by navigating to another page within the current browser tab, or by completely closing the tab.",
-        "WindowUnloadTriggerName": "Window Unload",
+        "WindowLeaveTriggerHelp": "This trigger is triggered when the user is possibly about to leave the website and moves their mouse outside of the current web page, e.g. towards the web-browser URL-address line or any other buttons. This can be useful to keep your visitor engaged and not lose the visitor, e.g. by showing an exit popup. You may be also interested in the \"Window Unload\" trigger which is triggered as soon as the user has actually left the current web page.",
+        "WindowLeaveTriggerName": "Window leave",
+        "WindowLoadedTriggerDescription": "Triggered when the web page is fully loaded including all images etc.",
+        "WindowLoadedTriggerHelp": "This stage of loading a web page, is when the page is fully loaded according to the web browser. In most cases this means that also all images are loaded, and all styles are applied etc. This trigger is always triggered at some point after the \"DOM Ready\".",
+        "WindowLoadedTriggerName": "Window loaded",
+        "WindowUnloadTriggerDescription": "Triggered just before the web-browser window is closed or when the user navigates to a different page.",
+        "WindowUnloadTriggerHelp": "This trigger identifies the event when a user is actually closing the current page compared to the \"Window Leave\" trigger which triggers when the user is about to leave your page. Either by navigating to another page within the current web browser tab, or by completely closing the tab.",
+        "WindowUnloadTriggerName": "Window unload",
         "ZendeskChatTagDescription": "Reach your customers, instantly via web, mobile, and messaging.",
-        "ZendeskChatTagHelp": "This tag allows you to add the Zendesk Chat form to your website.",
+        "ZendeskChatTagHelp": "This tag allows adding the Zendesk Chat form to your website.",
         "ZendeskChatTagName": "Zendesk Chat (formerly Zopim)",
-        "SettingElementVisibilityObserveDomChangesTitle": "Observe DOM Changes",
```

Comment:
@comradekingu Why was this removed?

Reply:
Don't really know, but adding it back in.